.github/workflows/backup-db.yml

```diff
@@ -78,3 +88,45 @@ jobs:
         backup-file: ${{ env.BACKUP_FILE }}.sql
         db-server-name: ${{ inputs.db-server }}
         slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+
+    - name: Create local sanitised database
+      run: |
+        createdb ${DATABASE_NAME} && psql -f ${{ env.BACKUP_FILE }}.sql -d ${DATABASE_NAME}
```

Comment:
The prod dump is a compressed file (.sql.gz), so I think it will have to be uncompressed. So something like "createdb ${DATABASE_NAME} && gzip -d --to-stdout ${{ env.BACKUP_FILE }}.sql.gz | psql -d ${DATABASE_NAME}"

Some of the steps could be merged as per https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/.github/workflows/database.yaml#L109

Reply:
Yes! Thanks for this